sormas-api/src/main/java/de/symeda/sormas/api/dashboard/EpiCurveGrouping.java

```diff
@@ -0,0 +1,7 @@
+package de.symeda.sormas.api.dashboard;
+
+public enum EpiCurveGrouping {
```

Comment:
Please remove the enum from the UI and use this new enum instead (don't forget to add the toString method here)